Frameworks/BlockchainCertificates/Certificate.swift

```diff
@@ -382,8 +398,10 @@ private enum MethodsForV1_2 {
                 return nil
         }
         
-        return Recipient(givenName: givenName,
-                         familyName: familyName,
+        let name : String = givenName + " " + familyName
+
+        
+        return Recipient(name: name,
```

Comment:
I'm a little concerned about these changes for v1.1 and v1.2. I figured we would leave these bits of code untouched, and only modify methods on a new `MethodsForV2` enum. We could support new constructors for this new interface.